libs/blocks/src/lib/cta/cta-left/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Heading, Section, Text, qtMerge } from '@deriv/quill-design';
+
+export type CTABlockProps = {
+  content?: React.ReactNode;
+  children?: React.ReactNode;
+  title: React.ReactNode | string;
+  description?: React.ReactNode | string;
+  className?: string;
+  variant?: 'content-left' | 'content-right';
+};
+export const CTALeft = ({
+  content,
+  children,
+  title,
+  description,
+  variant = 'content-left',
+}: CTABlockProps) => {
+  return (
+    <Section className=" py-general-4xl">
+      <div
+        className={qtMerge(
+          variant === 'content-left' ? 'lg:flex-row' : 'lg:flex-row-reverse ',
+          ' relative isolate flex h-full min-h-[496px] justify-end gap-gap-lg  max-lg:flex-col md:justify-center lg:mx-auto  lg:max-w-[1232px] ',
+        )}
+      >
+        {content && (
+          <>
+            <div className="absolute inset-50 z-10 bg-cta-gradient lg:bg-none"></div>
+            <div className="-z-10 h-full w-full overflow-hidden  object-cover object-top max-lg:absolute lg:relative ">
```

Comment:
Why are we using `max-lg`?

Reply:
```suggestion
            <div className="-z-10 h-full w-full overflow-hidden  object-cover object-top absolute lg:relative ">
```

Reply:
Hi @meenakshi-deriv can you please help to check @hasan-deriv's question ?